src/views/clusteroverview/SettingsTab/ClusterTab/components/GeneralSettings/MemoryDensity/MemoryDensity.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { FC, useCallback } from 'react';
+
+import HyperConvergedModel from '@kubevirt-ui/kubevirt-api/console/models/HyperConvergedModel';
+import SectionWithSwitch from '@kubevirt-utils/components/SectionWithSwitch/SectionWithSwitch';
+import { HyperConverged } from '@kubevirt-utils/hooks/useHyperConvergeConfiguration';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { k8sPatch } from '@openshift-console/dynamic-plugin-sdk';
+import { Skeleton } from '@patternfly/react-core';
+
+import { useIsAdmin } from '../../../../../../../utils/hooks/useIsAdmin';
```

Comment:
can this be `@utils/hooks/useIsAdmin`? or something similar?

Reply:
sure, auto import gone wrong lol